tasks/init_debian.yml

```diff
@@ -13,26 +13,26 @@
 
 - name: Configure CernVM apt repository
   apt_repository:
-    filename: "cernvm.list"
```

Comment:
any reason for changing filename? This will then generate duplicate entries in apt for sites which were using this role previously for long-term nodes.

Reply:
You are right. I revert it.